CHANGELOG.rst

```diff
@@ -2,30 +2,200 @@
 Release Notes
 =============
 
-.. _Release Notes_7.22.1:
```

Comment:
We should add things not remove some. We're deleting the 7.22.1 changelog righ now

Reply:
Fixed